packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { randomId } from '@rentpath/react-ui-utils'
+import Star from './SampleSvgs/Star'
+
+export default class RatingBar extends Component {
+  static propTypes = {
+    theme: PropTypes.object,
+    ratingScale: PropTypes.number,
+    rating: PropTypes.number,
+    total: PropTypes.string,
+    RatingItem: PropTypes.node,
+  }
+  static defaultProps = {
+    theme: {},
+    RatingItem: Star,
+  }
+
+  get ratingItems() {
+    const {
+      ratingScale,
+      rating,
+      RatingItem,
+      ...props
+    } = this.props
+
+    return [...Array(ratingScale).keys()].map(index => (
+      <RatingItem
+        key={randomId('rating')}
+        id={randomId('rating-item')}
```

Comment:
`index` cannot be used here due to the `LinearGradient` setting this as `id` and the `id` being referenced by the `fill={`url(#${id})`}` in the `Star`. You have a chance for id collisions if we do this. Maybe we can reference differently in the `fill` to fix the issue.